crates/pkgcraft-tools/tests/pkg/source.rs

```diff
@@ -35,35 +36,42 @@ fn invalid_pkgs() {
     temp.create_ebuild_from_str("cat/b-1", &data).unwrap();
     let path = temp.path();
 
-    // dep restriction
-    cmd("pk pkg source")
-        .args(["-r", path.as_str()])
-        .arg("cat/a-1")
-        .assert()
-        .stdout("")
-        .stderr(lines_contain(["invalid pkg: cat/a-1::test: line 4: die: error: msg"]))
-        .failure()
-        .code(1);
+    fn dep_restriction(path: &Utf8Path) {
+        cmd("pk pkg source")
```

Comment:
Would there be a way to modify assert_cmd::Command to support better disambiguation on failure? It's used a lot in various tests and this only fixes it in one specific place.

Reply:
we might ask upstream, the panic is inside their code. I can look into that later.

Reply:
We can use https://docs.rs/assert_cmd/latest/assert_cmd/assert/struct.Assert.html#method.append_context but it is a bit pointlessly verbose.

Reply:
as it is now the problems are mainly in nextest not reporting the line out of box, anyway that patch is spun off and this set could possibly land since makes the life simpler all in all ^^